.drone.yml

```diff
@@ -12,6 +12,24 @@ pipeline:
     commands:
       - apk update && apk add bash
       - bash -c 'shopt -s globstar; shellcheck **/*.sh'
+  setup-e2e:
+    image: docker.io/hashicorp/terraform:0.11.10
+    secrets: [ digitalocean_token ]
+    commands:
+      - ./e2e/drone_setup.sh
+  e2e:
+    image: ruby:2.5
+    commands:
+      - gem install bundler -Nf
+      - bundle install --path bundler
```

Comment:
These should probably use `--without test development`.

It could also do `gem build pharos-cluster.gemspec && gem install pharos-cluster-*.gem` and skip bundling altogether.

Reply:
Now uses the gem.